js/script.js

```diff
@@ -1,21 +1,21 @@
-class Stopwatch {
-  constructor(display) {
-    this.running = false;
-    this.display = display;
-    this.reset();
-    this.print(this.times);
+class Stopwatch extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      running: false,
+      miliseconds: 0,
+      seconds: 0,
+      minutes: 0,
+      results: []
+    }
   }
 
   addToList() {
-    let list = document.querySelector('.results');
-    let li = document.createElement('li');
-    li.setAttribute('class', 'result');
-    li.appendChild(document.createTextNode(this.format(this.times)));
-    list.appendChild(li);
+    this.state.results.push(this.format());
   }
 
   clearList() {
-    document.querySelector('.results').innerHTML = '';
+    this.setState({results: []});
```

Comment:
which method of clearing array is better in your opinion? :)

https://stackoverflow.com/questions/1232040/how-do-i-empty-an-array-in-javascript